test/test_autograd.py

```diff
@@ -4817,7 +4817,9 @@ def fn(*inputs):
                                         'broadcast_all' in test_name or
                                         'atanh' in test_name or
                                         'acosh' in test_name or
-                                        'asinh' in test_name)
+                                        'asinh' in test_name or
+                                        'abs_complex' in test_name or
```

Comment:
Is this "in" or `test_name is 'abs_complex'` ?

Reply:
`in` to disable these tests on both `cpu` and `cuda`